src/components/Book/Book.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => (
+  <tr>
+    <td>{ book.id }</td>
+    <td>{ book.title }</td>
+    <td>{ book.category }</td>
+  </tr>
+);
+
+Book.propTypes = {
+  book: PropTypes.objectOf(PropTypes.string),
+};
+
+Book.defaultProps = {
+  book: {},
+};
+
+export default Book;
```

Comment:
- `Book` component should be placed in `src/components` directory not `src/component/Book`